aws-rds-dbcluster/src/test/java/software/amazon/rds/dbcluster/CreateHandlerTest.java

```diff
@@ -169,55 +169,39 @@ public void handleRequest_CreateDbCluster_ServerlessV2ScalingConfiguration() {
     }
 
     @Test
-    public void handleRequest_CreateDbCluster_AccessDeniedTagging() {
+    public void handleRequest_CreateDbCluster_UnauthorizedTaggingOperation() {
         when(rdsProxy.client().createDBCluster(any(CreateDbClusterRequest.class)))
                 .thenThrow(
                         RdsException.builder()
+                                .message("Role not authorized to execute rds:AddTagsToResource")
                                 .awsErrorDetails(AwsErrorDetails.builder()
                                         .errorCode(ErrorCode.AccessDeniedException.toString())
                                         .build()
-                                ).build())
-                .thenReturn(CreateDbClusterResponse.builder().build());
-        when(rdsProxy.client().addRoleToDBCluster(any(AddRoleToDbClusterRequest.class)))
-                .thenReturn(AddRoleToDbClusterResponse.builder().build());
```

Comment:
Do we need to add a test for AccessDenied failure during CreateDBCluster/AddRoleToDBCluster operations?

Original test verified that AccessDenied during soft-fail tagging did not break the creation of the resource unintentionally. We might want to check that Acces denied during CreateDBCluster/AddRoleToDBCluster operations actually has the intended effect.

I think this is the case as well for RestoreToPointInTime and RestoreFromSnapshot scenarios

Reply:
Other tests are covering these scenario such as `AddRoleToDBCluster` scenario